bld/CLMBuildNamelist.pm

```diff
@@ -4137,6 +4143,24 @@ sub setup_logic_fates {
     }
 }
 
+#-------------------------------------------------------------------------------
+sub setup_logic_exice {
+  #
+  # excess ice streams
+  #
+  my ($opts, $nl_flags, $definition, $defaults, $nl) = @_;
+  my $use_exice = $nl->get_value( 'use_excess_ice'               );
+  if (defined($use_exice) && $opts->{'driver'} eq "nuopc" && value_is_true($use_exice)) {
+    add_default($opts, $nl_flags->{'inputdata_rootdir'}, $definition, $defaults, $nl, 'stream_meshfile_exice');
+    add_default($opts, $nl_flags->{'inputdata_rootdir'}, $definition, $defaults, $nl, 'stream_fldfilename_exice');
+    add_default($opts, $nl_flags->{'inputdata_rootdir'}, $definition, $defaults, $nl, 'stream_mapalgo_exice');
+  } elsif (defined($use_exice) && (! $opts->{'driver'} eq "nuopc") && value_is_true($use_exice)) {
```

Comment:
I like the test for nuopc driver here. We should add this check to the namelist testing. I can help with that.

Reply:
Since, we are scheduling to end support for MCT by Sep/30th there really isn't need to add a check for this, since it won't matter after that point. So I won't add the check that I thought I should add earlier.